assets/app/models/Site.js

```diff
@@ -0,0 +1,23 @@
+var Backbone = require('backbone');
+
+var SiteModel = Backbone.Model.extend({
+  defaults: {
+    'builds': [],
+    'repository': 'Repo goes here',
+    'engine': 'jekyll',
+    'branch': 'master'
+  },
+  urlRoot: '/v0/site'
```

Comment:
I'm open to any of these approaches. I like the idea of using the api header because it removes it as a concern across all of the models but I also like it in the URL for debugging purposes.

It is slightly redundant but we can refactor it pretty easily. Will do in #33